runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -504,7 +504,7 @@ static void AM_signal_long(gasnet_token_t token, void *buf, size_t nbytes,
 
 static void AM_priv_bcast(gasnet_token_t token, void* buf, size_t nbytes) {
   priv_bcast_t* pbp = buf;
-  chpl_memcpy(chpl_private_broadcast_table[pbp->id], pbp->data, pbp->size);
+  chpl_memmove(chpl_private_broadcast_table[pbp->id], pbp->data, pbp->size);
```

Comment:
This can't overlap.  The target is the address of a Chapel global variable and the source is within a GASNet-managed incoming message buffer.